job_runner/reporting/file.py

```diff
@@ -32,42 +37,56 @@ def __init__(self, sync_disc_timeout=10):
         self.node = socket.gethostname()
         self._sync_disc_timeout = sync_disc_timeout
 
-    def report(self, msg):
+    def report(self, msg: Message):
         job_status = {}
+
+        def job_status_insert(**kwargs) -> None:
+            for key, value in kwargs.items():
+                job_status[key] = value
+
         if msg.job:
-            index = msg.job.index
-            job_status = self.status_dict["jobs"][index]
+            logger.debug("Adding message job to status dictionary.")
+            job_status = self.status_dict["jobs"][msg.job.index]
 
         if isinstance(msg, Init):
+            logger.debug("Init Message Instance")
             self._delete_old_status_files()
             self._init_status_file()
             self.status_dict = self._init_job_status_dict(
                 msg.timestamp, msg.run_id, msg.jobs
             )
 
         elif isinstance(msg, Start):
+            logger.debug("Start Message Instance")
```

Comment:
It's not about the message, it's about the job… 
```suggestion
            logger.debug("Job {…} started")
```